themes/cardinal_service/lib/components/opportunity-row/opportunity_row.html.twig

```diff
@@ -1,6 +1,6 @@
 <div{{ attributes.addClass('su-opportunity-result') }}>
   <div class="flex-container su-card__contents">
-    <div class="{% if image %}flex-9-of-12{% endif %}">
+    <div class="{% if image|render|striptags('<img> <picture> <source>')|trim  %}flex-9-of-12{% endif %}">
```

Comment:
its best practice to also keep `<drupal-render-placeholder>` since it will keep cache mechanism functional.
see https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/templates/components/card/card.html.twig#L4

Be sure to do this any time you do `striptags`

Reply:
Thanks Mike! I'll make sure to keep that in mind for all the `striptags`